pkgs/games/fallout2-ce/default.nix

```
@@ -0,0 +1,49 @@
+{ cmake
+, lib
+, SDL2
+, stdenv
+, fetchFromGitHub
+, zlib
+}:
+
+stdenv.mkDerivation (finalAttr: {
+  pname = "fallout2-ce";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "alexbatalov";
+    repo = "fallout2-ce";
+    rev = "v${finalAttr.version}";
+    hash = "sha256-+N4jhmxBX6z48kaU0jm90OKhguHlggT3OF9uuyY0EV0=";
+  };
+
+  fpattern = fetchFromGitHub {
+    owner = "Loadmaster";
+    repo = "fpattern";
+    rev = "v1.9";
+    hash = "sha256-/QvMQCmoocaXfDm3/c3IAPyfZqR6d7IiJ9UoFKZTpVI=";
+  };
+
+  buildInputs = [ cmake SDL2 zlib ];
+  hardeningDisable = [ "format" ];
+  preConfigure = ''
+    substituteInPlace third_party/fpattern/CMakeLists.txt --replace "FetchContent_Populate" "#FetchContent_Populate"
+    substituteInPlace third_party/fpattern/CMakeLists.txt --replace "{fpattern_SOURCE_DIR}" "${finalAttr.fpattern}"
+    substituteInPlace third_party/fpattern/CMakeLists.txt --replace "$/nix/" "/nix/"
```

```suggestion
    substituteInPlace third_party/fpattern/CMakeLists.txt \
      --replace "FetchContent_Populate" "#FetchContent_Populate" \
      --replace "{fpattern_SOURCE_DIR}" "${finalAttr.fpattern}" \
      --replace "$/nix/" "/nix/"
```
Please move this to postPatch. Can we use our system libraries?